cdc/sink/mysql/mysql_syncpoint_store.go

```diff
@@ -238,7 +238,7 @@ func (s *mysqlSyncPointStore) SinkSyncPoint(ctx context.Context,
 				schemaName+"."+
 				syncPointTableName+
 				" WHERE changefeed like '%s' and created_at < (NOW() - INTERVAL %.2f SECOND)",
-			id.Namespace+"_"+id.ID,
+			id.ID,
```

Comment:
Should the cluster id be put into the conditions?

Reply:
yep

Reply:
thank for your remind~